modules/notification/base/null.go

```diff
@@ -58,14 +58,6 @@ func (*NullNotifier) NotifyUpdateComment(doer *models.User, c *models.Comment, o
 func (*NullNotifier) NotifyDeleteComment(doer *models.User, c *models.Comment) {
 }
 
-// NotifyDeleteRepository places a place holder function
-func (*NullNotifier) NotifyDeleteRepository(doer *models.User, repo *models.Repository) {
```

Comment:
why do you remove this?

```
./services/repository/repository.go:
notification.NotifyDeleteRepository(doer, repo)
```
its still used ?!?

Reply:
Just moved them with all repository notifier together. See https://github.com/go-gitea/gitea/pull/8573/files#diff-bfc9d877a9bf8dd799436270bb957949R103